controllers/keda/metrics_adapter_controller.go

```diff
@@ -108,42 +105,10 @@ func (r *MetricsScaledObjectReconciler) addToMetricsCache(namespacedName string,
 	scaledObjectsMetricsLock.Lock()
 	defer scaledObjectsMetricsLock.Unlock()
 	scaledObjectsMetrics[namespacedName] = metrics
```

Comment:
You should remove this one as well. Maybe the whole controller?

Reply:
lol, you are right again!

Reply:
done! I have updated the docs as well https://github.com/kedacore/keda-docs/pull/1132